sdcm/cluster.py

```diff
@@ -4877,7 +4877,7 @@ def reconfigure_scylla_monitoring(self):
             for db_node in self.targets["db_cluster"].nodes:
                 monitoring_targets.append(f"[{getattr(db_node, self.DB_NODES_IP_ADDRESS)}]:9180")
             monitoring_targets = " ".join(monitoring_targets)
-            if self.params.get("ip_ssh_connections") == "ipv6":
```

Comment:
It is related to monitoring only, not "mgmt" deployment. Why is it in the same commit?

Reply:
Moved to another commit